### DISCUSSION TITLE
Pytorch2 improvements?

### DISCUSSION THREAD

**Discussion by anhstar** [lshqqytiger/stable-diffusion-webui-directml#108] [General]
I have installed the latest version of this fork with pytorch2 support but did not notice any improvement vs the older version.
Im using a RX 6800 with 16 gig VRAM and on both version, I get around 1.5 it/s to 1.8 it/s
Windering if anyone found a way to improve the performance ? 
Thanks

**Comment by lshqqytiger** [comment]
There's no performance improvement.
I think performance-side things of PyTorch 2 are mostly related to SDKs like CUDA/ROCm/ipex.
Because DirectML is not a SDK, just a library, PyTorch is not compiled for/with it and then is not optimized for it.

**Reply by anhstar** [reply]
I see now, thanks for your answer :)
